### DISCUSSION TITLE
Question about Gm2 models

### DISCUSSION THREAD

**Discussion by xauspro** [storm-devs/storm-engine#432] [Q&A]
We are trying to work with models of the format .Gm2, which was used by Snail on the Storm engine, we need import and export so that we can work with these models in a blender, if you have a solution?

**Comment by Hammie** [comment]
HighStreet does not use this game engine, they might be using another "storm" engine, but it is not this one.

Their `gm2` files are also in no way structured like our `gm` files, so we cannot help you with this either.
